test/integration/default/build/s3.tf

```diff
@@ -0,0 +1,81 @@
+resource "random_id" "bucket_id" {
+  byte_length = 8
+}
+
+resource "aws_s3_bucket" "aws_demo_bucket" {
+  bucket        = "aws-demo-s3-bucket-${random_id.bucket_id.hex}"
+  acl           = "public-read"
+  force_destroy = true
+  region        = "us-east-1"
+  tags {
+    Name        = "aws_demo_bucket"
+    Environment = "prod"
+  }
+}
+
+output "s3_bucket_name" {
+  value = "aws-demo-s3-bucket-${random_id.bucket_id.hex}"
+}
+
+# add s3 bucket elements - pub
+
+resource "aws_s3_bucket_object" "public-read" {
+  bucket = "${aws_s3_bucket.aws_demo_bucket.id}"
+  acl    = "public-read"
+  key    = "public-pic-read.jpg"
+  source = "./data/public-pic.jpg"
+  etag   = "${md5(file("./data/public-pic.jpg"))}"
+
+  depends_on = ["aws_s3_bucket.aws_demo_bucket"]
+}
+
+# add s3 bucket elements - pub Authenticated Users only
+
+resource "aws_s3_bucket_object" "authenticated-read" {
+  bucket = "${aws_s3_bucket.aws_demo_bucket.id}"
+  acl    = "authenticated-read"
+  key    = "public-pic-authenticated.jpg"
+  source = "./data/public-pic.jpg"
+  etag   = "${md5(file("./data/public-pic.jpg"))}"
+
+  depends_on = ["aws_s3_bucket.aws_demo_bucket"]
+}
+# add s3 bucket elements - pri
+
+resource "aws_s3_bucket_object" "private" {
+  bucket = "${aws_s3_bucket.aws_demo_bucket.id}"
+  acl = "private"
+  key    = "private-pic.jpg"
+  source = "./data/private-pic.jpg"
+  etag   = "${md5(file("./data/private-pic.jpg"))}"
+
+  depends_on = ["aws_s3_bucket.aws_demo_bucket"]
+}
+
+# add s3 bucket policy elements
```

Comment:
Could you remove the bucket policy grants, and commit them on the bucket policy PR?